src/MixedFeed.php

```diff
@@ -1,48 +1,41 @@
 <?php
+
 namespace RZ\MixedFeed;
 
+use DateTime;
 use GuzzleHttp\Client;
+use GuzzleHttp\Exception\RequestException;
 use GuzzleHttp\Pool;
 use GuzzleHttp\Psr7\Response;
 use RZ\MixedFeed\Canonical\FeedItem;
 use RZ\MixedFeed\Exception\FeedProviderErrorException;
-use RZ\MixedFeed\MockObject\ErroredFeedItem;
 
 /**
  * Combine feed providers and sort them by date and time, descending or ascending.
  */
-class MixedFeed extends AbstractFeedProvider
```

Comment:
Extending AbstractFeedProvider didn't really make sense on the MixedFeed class. MixedFeed doesn't really behave like a provider.